### DISCUSSION TITLE
Markup jobposting

### DISCUSSION THREAD

**Discussion by Maga111** [schemaorg/schemaorg#3116] [Q&A]
Hi All!
I added a markup for jobposting, now want to add hiringOrganization, but don't know how to realize it perfectly, because they are categories/vacancies like Nanny, Housekeeper etc. which are hired by individuals.

Thanks in advance for the tips!

**Comment by thadguidry** [comment]
hiringOrganization indeed has only an expected type of Organization, but I would say is an omission of Person because it has taxId, vatId, and hasPOS and offerCatalog.  Go ahead and use a Person type and disregard any validator error on that I would say.

@danbri we probably should fix that in next release.  And for now link this Discussion to a new [issue #3144 ](https://github.com/schemaorg/schemaorg/issues/3144)

**Comment by rrlevering** [comment]
I feel like this should be solved by co-typing Person and Organization.  I think technically in most legal ways a person who is employing people are actually their own organization, aren't they?  I know in the US a person has to get a EIN for tax purposes so they are essentially running their own household business.

**Reply by thadguidry** [reply]
Yeap, it can definitely be solved alternatively with multi-types.
